src/GovUk.Education.ExploreEducationStatistics.Notifier.Model/PublicationNotificationMessage.cs

```diff
@@ -1,16 +1,27 @@
+#nullable enable
 using System;
 
 namespace GovUk.Education.ExploreEducationStatistics.Notifier.Model
 {
     public class PublicationNotificationMessage
     {
         public Guid PublicationId { get; set; }
-        public string Name { get; set; }
-        public string Slug { get; set; }
+        public string PublicationName { get; set; } = "";
+        public string PublicationSlug { get; set; } = "";
+
+        public string ReleaseName { get; set; } = "";
+        public string ReleaseSlug { get; set; } = "";
+
+        public bool Amendment { get; set; }
+        public string UpdateNote { get; set; } = "";
```

Comment:
Personal preference I guess but I think we use `string.Empty` in most places that I've seen for the default initialisation.